tools/docker/Dockerfile

```diff
@@ -4,6 +4,6 @@ FROM ubuntu:20.04
 RUN apt-get update -y
 RUN apt-get install -y gcc make xorriso qemu-utils
 # grub is a bit special in containers
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install grub2
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install grub2 python
```

Comment:
Not sure if we should explicitly make this Python3?

Reply:
My Docker had some issues with installing python3 packages. But I did not investigate.

Log:
```
Creating docker image
Sending build context to Docker daemon  3.501MB
Step 1/5 : FROM ubuntu:20.04
 ---> 1d622ef86b13
Step 2/5 : RUN apt-get update -y
 ---> Using cache
 ---> 5e47f6fd697e
Step 3/5 : RUN apt-get install -y gcc make xorriso qemu-utils
 ---> Using cache
 ---> 87a2609e7271
Step 4/5 : RUN DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install grub2 python3
 ---> Running in a757dbed6152
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  dmsetup file gettext-base grub-common grub-gfxpayload-lists grub-pc
  grub-pc-bin grub2-common libdevmapper1.02.1 libefiboot1 libefivar1 libexpat1
  libfreetype6 libfuse2 libmagic-mgc libmagic1 libmpdec2 libpng16-16
  libpython3-stdlib libpython3.8-minimal libpython3.8-stdlib mime-support
  os-prober python3-minimal python3.8 python3.8-minimal ucf xz-utils
Suggested packages:
  multiboot-doc grub-emu desktop-base console-setup fuse python3-doc
  python3-tk python3-venv python3.8-venv python3.8-doc binfmt-support
The following NEW packages will be installed:
  dmsetup file gettext-base grub-common grub-gfxpayload-lists grub-pc
  grub-pc-bin grub2 grub2-common libdevmapper1.02.1 libefiboot1 libefivar1
  libexpat1 libfreetype6 libfuse2 libmagic-mgc libmagic1 libmpdec2 libpng16-16
  libpython3-stdlib libpython3.8-minimal libpython3.8-stdlib mime-support
  os-prober python3 python3-minimal python3.8 python3.8-minimal ucf xz-utils
0 upgraded, 30 newly installed, 0 to remove and 4 not upgraded.
Need to get 9847 kB of archives.
After this operation, 48.6 MB of additional disk space will be used.
Ign:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpython3.8-minimal amd64 3.8.2-1ubuntu1.1
Get:2 http://archive.ubuntu.com/ubuntu focal/main amd64 libexpat1 amd64 2.2.9-1build1 [73.3 kB]
Ign:3 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python3.8-minimal amd64 3.8.2-1ubuntu1.1
Get:4 http://archive.ubuntu.com/ubuntu focal/main amd64 python3-minimal amd64 3.8.2-0ubuntu2 [23.6 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal/main amd64 mime-support all 3.64ubuntu1 [30.6 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal/main amd64 libmpdec2 amd64 2.4.2-3 [81.1 kB]
Ign:7 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpython3.8-stdlib amd64 3.8.2-1ubuntu1.1
Ign:8 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python3.8 amd64 3.8.2-1ubuntu1.1
Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 libpython3-stdlib amd64 3.8.2-0ubuntu2 [7068 B]
Get:10 http://archive.ubuntu.com/ubuntu focal/main amd64 python3 amd64 3.8.2-0ubuntu2 [47.6 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal/main amd64 libdevmapper1.02.1 amd64 2:1.02.167-1ubuntu1 [127 kB]
Err:1 http://security.ubuntu.com/ubuntu focal-updates/main amd64 libpython3.8-minimal amd64 3.8.2-1ubuntu1.1
  404  Not Found [IP: 91.189.88.142 80]
Get:12 http://archive.ubuntu.com/ubuntu focal/main amd64 dmsetup amd64 2:1.02.167-1ubuntu1 [75.6 kB]
Err:3 http://security.ubuntu.com/ubuntu focal-updates/main amd64 python3.8-minimal amd64 3.8.2-1ubuntu1.1
  404  Not Found [IP: 91.189.88.142 80]
Err:7 http://security.ubuntu.com/ubuntu focal-updates/main amd64 libpython3.8-stdlib amd64 3.8.2-1ubuntu1.1
  404  Not Found [IP: 91.189.88.142 80]
Get:13 http://archive.ubuntu.com/ubuntu focal/main amd64 libmagic-mgc amd64 1:5.38-4 [218 kB]
Err:8 http://security.ubuntu.com/ubuntu focal-updates/main amd64 python3.8 amd64 3.8.2-1ubuntu1.1
  404  Not Found [IP: 91.189.88.142 80]
Get:14 http://archive.ubuntu.com/ubuntu focal/main amd64 libmagic1 amd64 1:5.38-4 [75.9 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal/main amd64 file amd64 1:5.38-4 [23.3 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal/main amd64 ucf all 3.0038+nmu1 [51.6 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal/main amd64 xz-utils amd64 5.2.4-1 [82.5 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal/main amd64 gettext-base amd64 0.19.8.1-10build1 [50.2 kB]
Get:19 http://archive.ubuntu.com/ubuntu focal/main amd64 libfuse2 amd64 2.9.9-3 [82.0 kB]
Get:20 http://archive.ubuntu.com/ubuntu focal/main amd64 libpng16-16 amd64 1.6.37-2 [179 kB]
Get:21 http://archive.ubuntu.com/ubuntu focal/main amd64 libefivar1 amd64 37-2ubuntu2 [47.8 kB]
Get:22 http://archive.ubuntu.com/ubuntu focal/main amd64 libefiboot1 amd64 37-2ubuntu2 [38.1 kB]
Get:23 http://archive.ubuntu.com/ubuntu focal/main amd64 libfreetype6 amd64 2.10.1-2 [341 kB]
Get:24 http://archive.ubuntu.com/ubuntu focal/main amd64 grub-common amd64 2.04-1ubuntu26 [1853 kB]
Get:25 http://archive.ubuntu.com/ubuntu focal/main amd64 grub2-common amd64 2.04-1ubuntu26 [589 kB]
Get:26 http://archive.ubuntu.com/ubuntu focal/main amd64 grub-pc-bin amd64 2.04-1ubuntu26 [969 kB]
Get:27 http://archive.ubuntu.com/ubuntu focal/main amd64 grub-pc amd64 2.04-1ubuntu26 [125 kB]
Get:28 http://archive.ubuntu.com/ubuntu focal/main amd64 grub-gfxpayload-lists amd64 0.7 [3658 B]
Get:29 http://archive.ubuntu.com/ubuntu focal/main amd64 os-prober amd64 1.74ubuntu2 [20.1 kB]
Get:30 http://archive.ubuntu.com/ubuntu focal/universe amd64 grub2 amd64 2.04-1ubuntu26 [2572 B]
Fetched 5217 kB in 2s (2177 kB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python3.8/libpython3.8-minimal_3.8.2-1ubuntu1.1_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python3.8/python3.8-minimal_3.8.2-1ubuntu1.1_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python3.8/libpython3.8-stdlib_3.8.2-1ubuntu1.1_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python3.8/python3.8_3.8.2-1ubuntu1.1_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
The command '/bin/sh -c DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install grub2 python3' returned a non-zero code: 100
make: *** [dockerimage] Error 100```